templates/home.html

```diff
@@ -7,40 +7,107 @@
 
 {% endblock %}
 {% block content %}
-    <h2>Hello, {{ address }}</h2>
-    <h3>
-        Balance: {{ balance }}  $TN
-    </h3>
-    <h2>Extra fees: {{ extra_fees }}  $TN</h2>
-    <p>When your account is scripted, extra fees will need to be paid. When this isn't scripted, nothing will be
-        added.</p>
-    <h3>Transfer $TN:</h3>
-    <form id="send_form"><h2>Send:</h2>
-        <p>Fees: <input required id="send_fee_tn" maxlength="35"
-                        minlength="35" value="{{ (2000000 )/100000000+ extra_fees }}">
-        </p>
-        <p>Amount of $TN to send: <input required id="send_amount_tn" type="number"
-                                         max="{{ balance }}">
-        </p>
-        <p>Destination addr: <input required id="send_addr_tn" maxlength="35"
-                                    minlength="35"></p>
-        <p>
-            <button id="send_tn" type="button">Send
-                $TN
-            </button>
-        </p>
-    </form>
-    <h3>Transactions History</h3>
-    <button data-addr={{ address }} id="tx_load">Show tx history</button>
+   
+  <html>
```

Comment:
HTML and BODY tage included in the base already, do we really need it here?

Reply:
But base.html closes the body and html tag as well not sure whats correct we can remove opening and closing tags from home.html see what happens.